Arguments: 
  C:\Users\Tomohide\AppData\Local\Volta\tools\image\node\16.13.1\node.exe C:\Users\Tomohide\AppData\Local\Volta\tools\image\yarn\1.22.17\bin\yarn.js add @types/react-error-boundary

PATH: 
  C:\Users\Tomohide\AppData\Local\Volta\tools\image\yarn\1.22.17\bin;C:\Users\Tomohide\AppData\Local\Volta\tools\image\node\16.13.1;C:\Program Files\Volta\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\PROGRA~1\JPKI;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Azure Data Studio\bin;C:\Program Files\Amazon\AWSCLI\bin\;C:\Program Files\dotnet\;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\PuTTY\;C:\Users\Tomohide\AppData\Local\Volta\bin;C:\Users\Tomohide\AppData\Local\Microsoft\WindowsApps;;C:\Users\Tomohide\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\Tomohide\AppData\Local\Programs\EmEditor;C:\Program Files\JetBrains\PhpStorm 2021.1.2\bin;;C:\Program Files\Azure Data Studio\bin;C:\Program Files\JetBrains\WebStorm 2021.2.1\bin;;C:\Users\Tomohide\.dotnet\tools

Yarn version: 
  1.22.17

Node version: 
  16.13.1

Platform: 
  win32 x64

Trace: 
  Error: https://registry.yarnpkg.com/@types%2freact-error-boundary: Not found
      at Request.params.callback [as _callback] (C:\Users\Tomohide\AppData\Local\Volta\tools\image\yarn\1.22.17\lib\cli.js:67029:18)
      at Request.self.callback (C:\Users\Tomohide\AppData\Local\Volta\tools\image\yarn\1.22.17\lib\cli.js:140883:22)
      at Request.emit (node:events:390:28)
      at Request.<anonymous> (C:\Users\Tomohide\AppData\Local\Volta\tools\image\yarn\1.22.17\lib\cli.js:141855:10)
      at Request.emit (node:events:390:28)
      at IncomingMessage.<anonymous> (C:\Users\Tomohide\AppData\Local\Volta\tools\image\yarn\1.22.17\lib\cli.js:141777:12)
      at Object.onceWrapper (node:events:509:28)
      at IncomingMessage.emit (node:events:402:35)
      at endReadableNT (node:internal/streams/readable:1343:12)
      at processTicksAndRejections (node:internal/process/task_queues:83:21)

npm manifest: 
  {
    "dependencies": {
      "react-error-boundary": "^3.1.4"
    }
  }

yarn manifest: 
  No manifest

Lockfile: 
  # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
  # yarn lockfile v1
  
  
  "@babel/runtime@^7.12.5":
    version "7.16.7"
    resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.16.7.tgz#03ff99f64106588c9c403c6ecb8c3bafbbdff1fa"
    integrity sha512-9E9FJowqAsytyOY6LG+1KuueckRL+aQW+mKvXRXnuFGyRAyepJPmEo9vgMfXUA6O9u3IeEdv9MAkppFcaQwogQ==
    dependencies:
      regenerator-runtime "^0.13.4"
  
  react-error-boundary@^3.1.4:
    version "3.1.4"
    resolved "https://registry.yarnpkg.com/react-error-boundary/-/react-error-boundary-3.1.4.tgz#255db92b23197108757a888b01e5b729919abde0"
    integrity sha512-uM9uPzZJTF6wRQORmSrvOIgt4lJ9MC1sNgEOj2XGsDTRE4kmpWxg7ENK9EWNKJRMAOY9z0MuF4yIfl6gp4sotA==
    dependencies:
      "@babel/runtime" "^7.12.5"
  
  regenerator-runtime@^0.13.4:
    version "0.13.9"
    resolved "https://registry.yarnpkg.com/regenerator-runtime/-/regenerator-runtime-0.13.9.tgz#8925742a98ffd90814988d7566ad30ca3b263b52"
    integrity sha512-p3VT+cOEgxFsRRA9X4lkI1E+k2/CtnKtU4gcxyaCUreilL/vqI6CdZ3wxVUx3UOUg+gnUOQQcRI7BmSI656MYA==
